### DISCUSSION TITLE
Issue related to carrying out a transaction

### DISCUSSION THREAD

**Discussion by Forid703** [zkSync-Community-Hub/zksync-developers#237] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I made a transaction 14 ours ago but it has still not completed.
0x161ea06ad33ad10aa1c758bf38b4adf29e87081a442e380dfec36ac40c627e71

This is my transaction hash, plz check it out.

### What is the specific issue or error you're encountering?

0x161ea06ad33ad10aa1c758bf38b4adf29e87081a442e380dfec36ac40c627e71

This is the transaction hash, which i started to mint an nft from intract.io its been 14 hours but still the transaction is on its way, neither completed nor got cancelled.

### Can you share the error messages or logs you're receiving, if any?

0x161ea06ad33ad10aa1c758bf38b4adf29e87081a442e380dfec36ac40c627e71

I think transaction hash is enough.

### Have you made any recent changes to the contract before encountering this issue?

No

### Are there any external libraries or contracts that your contract interacts with?

No

### Can you provide the relevant portions of your contract code where the issue is occurring?

0x161ea06ad33ad10aa1c758bf38b4adf29e87081a442e380dfec36ac40c627e71

### Have you tried to isolate the problem, and if so, what were the results?

No

### What steps have you already taken to try to resolve the issue?

Nothing i could do

**Comment by 0xsisyfos** [comment]
Hello @Forid703 

Thank you for reaching out.

Your transaction seems totally normal (https://explorer.zksync.io/tx/0x161ea06ad33ad10aa1c758bf38b4adf29e87081a442e380dfec36ac40c627e71). You just need a bit of patience for it to be executed on Ethereum.

You can read more on the transaction status in our documentation: https://era.zksync.io/docs/reference/concepts/transactions.html#transaction-statuses

Best,
